tock/tock/context_processors.py

```diff
@@ -23,3 +24,16 @@ def tock_request_form_url(request):
     return {
         'x_tock_change_request_form_url': settings.TOCK_CHANGE_REQUEST_FORM,
     }
+
+def user_attendance(request):
+    response = {
+        'x_tock_user_is_late': None
+    }
+
+    try:
+        udata = UserData.objects.get(user=request.user)
+        response['x_tock_user_is_late'] = udata.is_late()
+    except UserData.DoesNotExist:
+        response['x_tock_user_is_late'] = False
+
+    return response
```

Comment:
I agree with @tbaxter-18f - there should be a way to get this information off of `request.user` or `request.user.user_data` directly, which would keep things simpler and obviate the need for an extra context processor. :-)